### DISCUSSION TITLE
Windows App SDK 1.0 Experimental (1.0.0-exp) is available

### DISCUSSION THREAD

**Discussion by dayman40** [microsoft/WindowsAppSDK#1203] [Announcements]
## 1.0 Experimental (1.0.0-exp)

We've just released our 1.0 experimental release of Windows App SDK! This is the latest release of the experimental channel that supports all [experimental channel features](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/release-channels#features-available-by-release-channel). This release includes bug fixes and provides access to non-stable APIs and features for WinUI3, Push notifications, Windowing, and Deployment of unpackaged apps. 

Check all our new and updated features, limitations, and known issues for this release on our full [release notes here](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/experimental-channel#version-10-experimental-100-experimental1).

## What's next
To keep up with progress being made on Windows App SDK, please see our [feature roadmap](https://github.com/microsoft/ProjectReunion/blob/main/docs/roadmap.md) which gets updated regularly. We also give monthly updates on our Windows App SDK Community Calls, where you can directly ask the team any questions you have. 

Attached are the Windows App SDK Installer and MSIX packages needed to test deployment of unpackaged apps or you can find it [here](https://download.microsoft.com/download/b/8/b/b8b4508e-ea29-4381-85d9-69b824151f8d/Microsoft.WindowsAppSDK.Redist.1.0.0-exp.zip). 

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.0.0-exp'>1.0 Experimental (1.0.0-exp)</a>.</em>

**Comment by riverar** [comment]
~~@dayman40 "release notes here" link is broken. "experimental channel features" link is broken.~~

**Reply by dayman40** [reply]
Hi @riverar, it was updated! Please check it again now and you'll find it working. Thank you.

**Reply by riverar** [reply]
@dayman40 Works great now, thanks!

**Reply by shelllet** [reply]
I have been waiting for this moment for a long time.

**Reply by shelllet** [reply]
I am disappointed with `Winui3- Not available for unpackaged desktop`

**Reply by MarkIngramUK** [reply]
Does an "unpackaged desktop app" include unpackaged apps that bootstrap the framework?

**Reply by shelllet** [reply]
I don't really understand the framework package, in short, "copy -> run". The deployment is simple and the same as ordinary win32.

`misx` need to set computer  to develop mode. this is complex for end user.

**Reply by riverar** [reply]
> Does an "unpackaged desktop app" include unpackaged apps that bootstrap the framework?

Yep!

**Comment by Alovchin91** [comment]
I'm confused now. Is WinUI 3 going to be supported for unpackaged desktop apps in 1.0?

**Reply by ghost1372** [reply]
according to the [roadmap](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/roadmap.md#unpackaged-desktop-apps) yes

**Reply by andrewleader** [reply]
Yes, WinUI 3 is planned to be supported for unpackaged desktop apps in 1.0. Just because it didn't make our first experimental release doesn't mean it isn't making the release all together 😊

**Comment by juanpaexpedite** [comment]
Hi, I have just updated VS 2020 to preview 3, and updated and installed all necessary and 0.8 works pretty well but for 1.0 blank app packaged with WAP (WinUI3 in desktop) I have the following issue:
![image](https://user-images.githubusercontent.com/5947026/129090825-64c37522-7c77-4e63-98d0-170a9783d5e2.png)

I do not know if it is a bug from the source because how is it possible does not exist InitializeComponent.
Does anyone have the same issue?
